CAIPs/caip-261.md

```diff
@@ -0,0 +1,466 @@
+A validity limitation can be introduced with the fields `validFrom` and `validUntil`.
+
+```json
+"@context": ["https://www.w3.org/2018/credentials/v1"],
+"type": ["VerifiableCredential", "PeerTrustCredential"],
+"issuanceDate": "2024-02-15T07:05:56.273Z",
+"issuer": "did:pkh:eip155:1:0x44dc4E3309B80eF7aBf41C7D0a68F0337a88F044",
+"validFrom": "2024-01-01T19:23:24Z",
+"validUntil": "2025-01-01T19:23:24Z",
+```
+
+##### On-chain assertions vaidity period
+Most on-chain assertion storages manage expiration times natively without requiring changes to the overall schema.
+
+### Trust Management
+
+Initially, issued trust assertions should be properly persisted to ensure their availability according to the use-case.
+Subsequently, as trust between peers evolves over time, this proposal outlines a method for managing the entire lifecycle of these assertions, encompassing their creation, update, and eventual revocation.
+
+#### Trust Persistance
+
+Trust Assertions issued by peers compose into a verifiable trust graph that can be consumed for trustless reputation-based use cases.
+To fulfill its purpose, these assertions SHOULD be persisted using mechanisms that provide these properties relative to the given use-case requirements such as Security, Data integrity, Data ownership, Interoperabiltiy, Composability, Scalability, Censorship resistance, Sustainability...
+
+#### Trust Update
+
+When an issuer needs to update a trust assertion, they simply generate a new assertion containing the revised information.
+This new assertion, when issued, MUST supersede any previous assertions of the same type, issued by the same entity, and pertaining to the same subject.
+
+The new assertion COULD refer to the overwritten assertion by adding its identifier in the top level of the data structure
+```json
+    ""previousVersion":": "ipfs://QmcwYEnWysTyepjjtJw19oTDwuiopbCDbEcCuprCBiL7gt",
+```
+
+#### Trust Revocation
+
+When an issuer decides to revoke a previously issued trust assertion without replacing it, they should generate a new assertion that includes a `credentialStatus` attribute, with a `statusPurpose` set to `revocation`.
+This new assertion should explicitly reference the assertion being revoked.
+By doing so, the system and its participants can easily identify the revocation status of any assertion, ensuring the trust graph remains accurate and up-to-date with the current trust relationships.
+
+**Revocation of trust assertion:**
+
+```json
+"@context": ["https://www.w3.org/2018/credentials/v1"],
+"type": ["VerifiableCredential", "PeerTrustCredential"],
+"issuanceDate": "2024-02-29T14:31:56.273Z",
+"issuer": "did:pkh:eip155:1:0x44dc4E3309B80eF7aBf41C7D0a68F0337a88F044",
+"credentialStatus": {
+    "id": "ipfs://QmcwYEnWysTyepjjtJw19oTDwuiopbCDbEcCuprCBiL7gt",
+    "type": "CredentialStatus",
+    "statusPurpose": "revocation",
+},
+```
+##### On-chain assertions revocation

```

Comment:
```suggestion

##### On-chain assertions revocation
```

sorry i've become obsessive about linting markdown lately because it sometimes displays wrong in jekyll if you don't have the newlines just right